src/util/MapPolyfill.ts

```diff
@@ -20,7 +20,7 @@ export class MapPolyfill {
     return this;
   }
 
-  delete(key): boolean {
+  delete(key: any) {
```

Comment:
I don't remember explicitly removing it, I think it was added recently.  I just didn't combine it in the merge when I saw it.